pjsip-apps/src/samples/streamutil.c

```diff
@@ -820,9 +868,7 @@ int main(int argc, char *argv[])
      * in this case are file_port and stream_port).
      */
```

Comment:
> > Already revert stream.c, when info->param is NULL, keep old logic ^_^
> 
> If the old logic only applies when `stream_info->param` is `NULL`, the risk of behavioural change will still be there because PJSUA-LIB populates the stream info using `pjmedia_stream_info_from_sdp()` which initiates `stream_info->param` using `pjmedia_codec_mgr_get_default_param()` so it won't be `NULL`.

You are right. stream.c has reverted to origin,  and streamutil opus custom settings implemented use a temp codec info